pandas/io/formats/css.py

```diff
@@ -27,12 +27,46 @@ def expand(self, prop, value: str):
     return expand
 
 
+def _border_expander(side: str = ""):
+    if side != "":
```

Comment:
can you add a doc-string & types on the return (+1 if you can also do this for `_side_expander`)